CareKitUI/CareKitUI/iOS/Link/LinkButton.swift

```diff
@@ -120,13 +120,21 @@ private struct InAppContent: View {
         if let url = link.url {
             switch link {
             case .website:
-                SafariView(url: url)
+                webView(withURL: url, link: link)
                     .edgesIgnoringSafeArea(.bottom)
             default:
                 fatalError("Link type does not support in-app content")
             }
         }
     }
+
+    func webView(withURL url: URL, link: LinkItem) -> some View {
```

Comment:
Created this method because some of the OS's don't support UIKit for `SafariView` or it didn't work properly (visionOS using Canvas). Let me know if you have an improvement here.